qt/scientific_interfaces/ISISReflectometry/GUI/Save/SavePresenter.cpp

```diff
@@ -42,6 +42,10 @@ void SavePresenter::acceptMainPresenter(IBatchPresenter *mainPresenter) {
   m_mainPresenter = mainPresenter;
 }
 
+void SavePresenter::notifySettingsChanged() {
+  m_mainPresenter->notifySettingsChanged();
```

Comment:
I don't think we can call this here because I think it will end up resetting the state of rows in the table. The Save tab is a bit different to the other tabs, because if we're just changing something like the filter criteria in the search box, this doesn't affect the reduction configuration, so therefore shouldn't reset the rows' state. However, I don't think this is called at the moment anyway so this bug is not showing up!